src/applications/proxy-rewrite/tests/health.integration.spec.js

```diff
@@ -0,0 +1,44 @@
+const puppeteer = require('puppeteer');
+
+describe('jest-image-snapshot usage with an image received from puppeteer', () => {
+  let browser;
+  jest.setTimeout(20000);
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+  });
+
+  it('compares screenshots of elements', async () => {
+    const page = await browser.newPage();
+    page.setViewport({ width: 1024, height: 800 });
```

Comment:
maybe pull the viewport size objects up a level so that can be shared? Or ideally move them to a helper file since I'm sure we'd want to be able to reuse them across future tests.

Reply:
Good idea, updated.